tools/generator/filters/python.py

```diff
@@ -60,3 +81,30 @@ def python_to_cxx(value: str) -> str:
         )
     else:
         return 'python_to_cxx<PyObject*, {}>'.format(cxx_type(value))
+
+
+@register_filter
+def get_python_import(functions) -> str:
```

Comment:
You should make this a function with pass_context instead. Then you can just look up the function dict in the context. It's cleaner and if other types start requiring imports, we won't have to change the templates.

Reply:
Well the point here is that we only need the imports for what is declared in `functions`. Because `_api.pyi` only declares `functions`. If other types need imports in `_api.pyi`, it’s a good think that we need to change the templates.

Keeping the function as is, we don’t have to modify it to reuse it in an other template, that would only need the imports for one single function.

But in both cases it’s speculation of what would be other uses. I don’t think it matters much.